README.md

```diff
@@ -1 +1,24 @@
-# Changes Browser [![Build Status](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser.svg?branch=team09)](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser)
+# ChangesReloaded [![Build Status](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser.svg?branch=team09)](https://travis-ci.org/HPI-SWA-Teaching/ChangesBrowser)
+
+ChangesReloaded is a project developed in context of SWT 2020 based on the Changes Browser project from 2017. It is supposed to simplify the old tool.
+
+## Installation
+In order to use our tool, you have to install it in your squeak image. You can either do that by using the Git Browser or by using Monticello.
+
+### Git Browser
+- Open the Git Browser in your squeak image (`Tools>Git Browser` or `Apps>Git Browser`).
```

Comment:
For telling users where to click, we could use the cool <kbd>button feature</kbd> of GitHub-flavored Markdown (as well as Unicode Arrows →):

```suggestion
- Open the Git Browser in your squeak image (<kbd>Tools</kbd> → <kbd>Git Browser</kbd> or <kbd>Apps</kbd> → <kbd>Git Browser</kbd>).
```

Here's how it would look:
- Open the Git Browser in your squeak image (<kbd>Tools</kbd> → <kbd>Git Browser</kbd> or <kbd>Apps</kbd> → <kbd>Git Browser</kbd>).